src/extension.ts

```diff
@@ -526,14 +528,17 @@ class NodeRepl extends EventEmitter {
         return code.replace(regex, (str, par, name) => {
             try {
                 if(require(name)) {
-                    return name;
```

Comment:
I can't mine or this to work. It resolves Debug mode but not Runtime. Think we should go absolute paths all the way.

Alternative is using Object.defineProperty in repl.context with a value of `require(name)`

Reply:
```suggestion
                if(require(name)) {
```
I think that this line is using to test the module whether is Node.js api or not. I pull request #15 to fix cannot import Node.js api.